setup.py

```diff
@@ -45,13 +46,14 @@ def get_description():
     version=get_version(),
     description='Compile SCSS files to valid Qt stylesheets.',
     long_description=get_description(),
+    long_description_content_type='text/markdown',
     author='Yann Lanthony',
     author_email='https://github.com/yann-lty',
     maintainer='Dan Bradham',
```

Comment:
I think this should be

    maintainer='Dan Bradham and the Spyder Project contributors'

Reply:
It may make more sense to switch it to "the Spyder Project Contributors" at this point. #33

Reply:
Ok, no problem with that.